server/user/userServices.js

```diff
@@ -0,0 +1,38 @@
+const config = require('../config.json');
+// const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs');
+const db = require('../helpers/db');
+const User = db.User;
+
+
+module.exports = {
+    createAccount,
+    login
+}
+
+
+async function createAccount(userParam) {
+
+    //if (await User.findOne({ username: userParam.username })){
+    //    throw 'Username "' + userParam.username + '" is already taken';
+    //} it's if I'll need to identificate users 
+    const user = new User(userParam);
+    if (userParam.password) {
+        user.hashPassword = bcrypt.hashSync(userParam.password, 10);
+    }
+
+    return await user.save();
+}
+async function login({email, password}) {
```

Comment:
leave a space in 1 line between functions.

Reply:
Done